src/utils/perfherder/chartJs/getPerfherderData.js

```diff
@@ -41,7 +41,7 @@ const getPerfherderData = async series => {
       `can not get data for ${JSON.stringify(series[0].seriesConfig)}`
     );
 
-  return perfherderFormatter(newData);
+  return perfherderFormatter(newData.filter(Boolean));
```

Comment:
If you approve this fix, I wasn't sure if `newData` should be filtered and saved first:
```
newData = newData.filter(Boolean);
return perfherderFormatter(newData);
```

or what I did.

Reply:
What you did was elegant enough, but it messed with the colors, so we will not use it.

Reply:
Yup, I completely agree, thank you for noticing the issue that would bring to us.